source/install/mattermost-omnibus.rst

```diff
@@ -81,6 +91,44 @@ Please be careful when using this feature, as making changes to the custom templ
 
 This feature is available from Mattermost Omnibus version 5.32.0.
 
+Omnibus Components
+------------------
+
+Mattermost Omnibus can be used to install and manage several external components.
+
+Jitsi
+^^^^^
+
+This component installs and configures `Jitsi <https://jitsi.org/>`_. Jitsi is an Open Source video conferencing software that can be used within Mattermost. 
+
+As Jitsi is a separate service to Mattermost, ensure that you have access to a different domain name than the one used for Mattermost pointing to the Omnibus server. This domain name is used to log into the Jitsi instance.
+
+To enable Jitsi on a running Omnibus installation, open the ``mmomni.yml`` configuration file and apply the following changes:
+
+- Set the ``jitsi_installed`` and ``jitsi_enabled`` configuration properties to ``true``.
+- Set the ``jitsi_fqdn`` property to the domain name you'll be using for Jitsi. It needs to be different from the domain used for ``fqdn``.
+- Set the ``jitsi_jvb_secret``, ``jitsi_focus_secret`` and ``jitsi_focus_password`` properties. Those are internal values used between different Jitsi modules to communicate in a secure fashion, and should have random values. You can use any secure password generator to obtain values for them.
+
+With all the values set, run ``sudo mmomni reconfigure`` for Omnibus to apply the changes and restart Mattermost. When the reconfigure process is finished, use ``jitsi_fqdn`` to access your new Jitsi instance.
+
+To integrate Jitsi and Mattermost, you can install and configure the `Mattermost Jitsi plugin <https://github.com/mattermost/mattermost-plugin-jitsi>`_.
+
+To uninstall the Jitsi component, open the ``mmomni.yml`` configuration file and set ``jitsi_installed`` and ``jitsi_enabled`` to ``false``. Then run ``sudo mmomni reconfigure`` for Omnibus to apply the changes and restart Mattermost.
+
+Monitoring
+^^^^^^^^^^
+
+This component installs and configures a `Grafana <https://grafana.com/>`_ instance with a set of default dashboards that monitor the health of your Mattermost Omnibus platform.
```

Comment:
This is not about grafana, this is about the Mattermost monitoring system. that includes prometheus and grafana, as the main pieces.

Besides, I would clearly point here, that this component is only usable with E20 license.

Reply:
Addressed in 03dfb61